src/pages/jisuOh/Main/CommentList/CommentList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Comment from '../Comment/Comment';
+
+class CommentList extends React.Component {
+  render() {
+    const { comments, deleteComment } = this.props;
+    if (!comments) {
+      return '';
+    } else {
+      return comments.map(content => {
+        return (
+          <Comment
+            key={content.id}
+            comments={content}
+            commentsArr={comments}
+            deleteComment={deleteComment}
+          />
+        );
+      });
+    }
```

Comment:
if else 구문은 depth가 깊어지는 형태를 띄고 있어서 가독성때문에 필요한 경우를 제외하고는 가급적 사용을 지양하고 있습니다.

함수는 return문을 만나면 종료됩니다. if문의 조건이 true라면은 return을 만나고 함수가 종료되어서 그 뒤에 라인은 읽지 않기에 굳이 else 구문을 쓸 필요가 없습니다!

그리고 빈 값을 리턴하고 싶을 때에는 빈 문자열을 리턴하는것보다는 아무겂도 없다라는 명확한 의미를 가지고 있는 `null` 을 리턴해주는게 좋습니다
```suggestion
    if (!comments) {
      return null
    }
    return comments.map(content => {
        return (
          <Comment
            key={content.id}
            comments={content}
            commentsArr={comments}
            deleteComment={deleteComment}
          />
        );
      });
    }
```

Reply:
그리고 Comment를 map만 돌리고 있는 역할을 하는 CommentsList라는 컴포넌트는 불필요해보입니다!
불필요하게 컴포넌트 depth가 깊어지는 것 같습니다!
상위 컴포넌트에서 바로  Comment를 map돌리는게 좋아보입니다!

Reply:
오 그렇군요! CommentList 삭제 완료했습니다!